src/com/trainee/geetanjali/java/DiagonalDifference.java

```diff
@@ -0,0 +1,44 @@
+package com.trainee.geetanjali.java;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class DiagonalDifference {
+    public static void main(String args[]) {
+        int s1 = 0, s2 = 0;
+        Scanner s = new Scanner(System.in);
+        System.out.println("Enter Size of a matrix:");
+        try {
+            int n = s.nextInt();
+            int a[][] = new int[n][n];
+            System.out.println("Enter the elements in  matrix");
+            for (int i = 0; i < n; i++) {
+                for (int j = 0; j < n; j++) {
+                    System.out.println("Enter a[" + i + "] [" + j + "]=");
+                    a[i][j] = s.nextInt();
+                }
+            }
```

Comment:
Insert Comments in the Code
// Reads the elements or Reads the input from the user and stores in the array